quickstart/resources/views/product/edit.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('layout.master')
+@section('script_product')
+
+@endsection
+@section('content')
+@component('layout.components.title')
+{{__('Edit page')}}
```

Comment:
```suggestion
    {{ __('Edit page') }}
```
tương tự các chỗ khác nhé